doc/doc.css

```diff
@@ -1,44 +1,226 @@
+    margin-right: 6 rem;
+}
+
+.header__links li i{
+    color: black;
+}
 
+.header__links li i:hover{
+    cursor: pointer;
+    /* color: var(--primary-dark-color) */
+    color: var(--secondary-color)
 }
 
-.hero-left{
-    grid-area: left;
+
+/* ****** Sidebar ******* */
+
+.sidebar-main{
+    position: relative;
+    grid-area: sidebar;
+    
+}
+
+.sidebar{
+    padding: 3rem;
+    padding-left: 4.5rem;
+    padding-right: 3rem;
+    position: sticky;
+    overflow-y: auto;
+    top: 7.5rem;
+    height: calc(100vh - 8.125rem);
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    
+}
+
+.sidebar::-webkit-scrollbar {
+    display: none;
+  }
+
+.sidebar__links{
     display: flex;
     flex-direction: column;
+    align-items: center;
+    gap: 1rem;
+    position: relative;
+}
+
+.sidebar__links::after{
+    content: "";
+    position: absolute;
+    border-right: 2.5px solid var(--primary-color);
+    top: -14rem;
+    right: -0.9rem;
+    height: 145%;
+}
+
+.sidebar h3{
+    display: grid;
     justify-content: center;
-    align-items: flex-start;
-    width: 40vw;
- 
+    
 }
 
-.hero-left > a {
-    width: 18rem;
-    height: 4.5rem;
+.sidebar a{
+    width: 100%;
+    font-weight: 500;
     font-size: 1.8rem;
-    text-align: center;
-    line-height: 24px;
+    color: var(--dark-color);
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
+    padding: 0.5rem;
+    display: grid;
+    justify-content: center;
+}
+
+.sidebar li{
+    list-style: none;
+}
+
+.sidebar a:hover{
+    color: var(--primary-light-color);
+}
+
+/* ****** scroll margin ***** */
+ #alert-section,
+ #avatar-section,
+ #badge-section,
+ #button-section,
+ #card-section,
+ #image-section,
+ #input-section{scroll-margin-block: 10rem;}
+
+/* ******* container ******* */
+
+.container{
+    padding: 1rem 2rem;
+    padding-left: 4rem;
+    top: 15rem;
+}
+
+.component-description{
+    margin: 2.2rem 0 3rem 0;
+}
+
+.component-description > h2{
+    color: var(--primary-color);
+    position: relative;
+}
+
+.component-example{
+    margin-top: 5rem
+}
+
+.component-example > h3{
+    margin-bottom: 3rem;
+}
+
+.component-example__container{
+    display: flex;
+    border: 1px solid rgb(20 20 20 / 12%);
+    border-radius: 0 2rem;
+    padding: 2rem;
+    background-color: var(--xlight-blue);
+    justify-content: space-evenly;
+}
+
+
+
+/* ***** section break line ***** */
+
+.section-break-line{
+    margin-top: 6rem;
+    margin-bottom: 5rem;
+}
+
+/* **** Code snippet **** */
+
+.code-snippet{
+    margin: 1rem 0 3rem 0;
+    /* min-height: auto; */
+    height: 21rem;
+    background-color: var(--xlight-blue);
+    border-radius: 0 2rem;
+    border: 1px solid rgb(20 20 20 / 12%);
+    /* overflow-y: auto; */
+    padding: 1rem;
+    padding-top: 3rem;
+    padding-bottom: 3rem;
+    display: flex;
+    /* flex-direction: column; */
+    /* justify-content: space-evenly; */
+    /* align-items: center; */
+    flex-wrap: wrap;
```

Comment:
Here also remove comments if not required

Reply:
Done